alternatives/entity-texture-features.mjs

```diff
@@ -2,14 +2,14 @@ import Mod from "../build_src/mod.mjs";
 
 export default new Mod(
 	"Entity Texture Features",
-	"Traben_0",
+	"Traben",
 	"A Fabric mod implementing randomized & emissive texture support for mobs set by the resourcepack. Fully compatible with the Optifine format & Sodium.",
 )
 .icon("https://cdn.modrinth.com/data/BVzZfTc1/icon.png")
-.add_version({ loader: ["fabric", "quilt"], v: [16, 17, 18, 19] }, { loader: ["forge"], v: [18] })
+.add_version({ loader: ["quilt"], v: [18, 19] }, { loader: ["fabric"], v: [16, 17, 18, 19] }, { loader: ["forge"], v: [16, 17, 18, 19] })
```

Comment:
If Fabric and Forge equivalents have the same supported version, why not merge the two back together since it has a different way to display that in the rendered page?